frontend/src/bundles/common/components/header/components/notifications/notification-component.tsx

```diff
@@ -0,0 +1,81 @@
+import { DataStatus } from '~/bundles/common/enums/enums.js';
+import {
+    useAppDispatch,
+    useAppSelector,
+    useCallback,
+    useEffect,
+    useHandleClickOutside,
+    useRef,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+import {
+    deleteNotification,
+    dismissNotification,
+    fetchNotifications,
+} from '~/bundles/notifications/store/actions.js';
+
+import { NotificationList } from './components/notification-list.js';
+import { NotificationBell } from './components/notifications-bell.js';
```

Comment:
Can we combine it like this: `import { NotificationList, NotificationIcon } from './components/components.js';` ?